docs/syncset.md

```diff
@@ -0,0 +1,94 @@
+# SyncSet
+
+## Overview
+
+`SyncSet` and `SelectorSyncSet` objects facilitate resource management (create, update, delete, patch) in hive-managed clusters.
```

Comment:
I'd add something like:
> To use `SyncSet` objects to manage resources, you must create them in the same namespace as the `ClusterDeployment` resource that they manage. If you want to manage resources in clusters that match a specific tag, use `SelectorSyncSet` objects instead. These objects apply changes to clusters in any namespace that match the `clusterDeploymentSelector` that you set.

> With both `SyncSets` and `SelectorSyncSets`, the individual resources and patches are reapplied when 2 hours have passed since their last reconciliation or if their contents are updated in the `SyncSet` or `SelectorSyncSets`.

> If you want to place the name that share a namespace with the `SyncSet`. `SyncSets` may contain a list of resource object definitions to create and a list of patches which are applied to existing objects.